app/views/shared/_fields_for.html.slim

```diff
@@ -0,0 +1,26 @@
+.multiple
+  div id=question.name
+  div id="error_#{f.object.name}_#{question.name}"
+
+  = f.fields_for question.name do |mf|
+    .multiple-wrapper class=(mf.index == 0 ? 'multiple-first' : 'multiple-not-first')
+      = mf.hidden_field :id
+      - question.subquestions.each do |subquestion|
+        .multiple-field.inline
+          - if subquestion.answers_options.empty?
+            = mf.text_field subquestion.name.to_sym
+          - else
+            .form-group
+              = mf.label subquestion.name.to_sym, class: 'form-label'
+              = mf.select subquestion.name.to_sym, select_values_for("#{f.object.name}.#{question.name}.#{subquestion.name}"), include_blank: true
```

Comment:
📓 I think this could be simplified, but there's still quite a lot of dependent models lying around.

Ideally in the future, the presenter for the question, should be able to respond directly to the `values_for` that question.